packages/@okta/vuepress-site/docs/concepts/auth-servers/index.md

```diff
@@ -0,0 +1,88 @@
+
+## What you can use an authorization server for
+
+You can use an authorization server to perform Single Sign-On (SSO) with Okta for your OpenID Connect (OIDC) apps, and to secure your own APIs and provide user authorization to access your web services.
+
+OIDC is used to authenticate users with a web app. The app uses the ID token that is returned from the authorization server to know if a user is authenticated and to obtain profile information about the user, such as their username or locale. OAuth 2.0 is used to authorize user access to an API. An app uses an access token to validate a user's identity. When using OIDC or OAuth, the authorization server authenticates a user and issues an ID token and/or an access token.
+
+> **Note:** You can't mix tokens between different authorization servers. By design, authorization servers don't have trust relationships with each other.
+
+## Authorization server types that are available in Okta
+
+Okta has two types of authorization servers: the Org Authorization Server and Custom Authorization Servers.
+
+### Org Authorization Server
+
+Every Okta org comes with a built-in authorization server called the Org Authorization Server (Org AS). The URL for the Org AS is `https://${yourOktaOrg}`.
+
+You use the Org AS to perform SSO with Okta or to get an access token for the Okta APIs. You can't customize this authorization server with regards to audience, claims, policies, or scopes. Additionally, the resulting access token's issuer is `https://${yourOktaOrg}`, which indicates that only Okta can consume or validate it. The access token can't be used or validated by your own applications.
+
+#### Org Authorization Server discovery endpoints
+
+The following discovery endpoints return OIDC or OAuth 2.0 metadata related to your Org AS. Clients can use this information to programmatically configure their interactions with Okta. Custom scopes and custom claims aren't returned.
+
+**OpenID:** `https://${yourOktaOrg}/.well-known/openid-configuration`
+
+**OAuth:** `https://${yourOktaOrg}/.well-known/oauth-authorization-server`
+
+### Custom Authorization Server
+
+You use a Custom Authorization Server to create and apply authorization policies to secure your APIs. An access token that is minted by a Custom Authorization Server is consumed by your APIs.
+
+Okta allows you to create multiple Custom Authorization Servers within a single Okta org that you can use to protect your own resource servers. Within each authorization server, you can define your own custom OAuth 2.0 scopes, claims, and access policies to support authorization for your APIs.
+
+#### Default Custom Authorization Server
+
+Okta provides a pre-configured Custom Authorization Server called `default`. It includes a basic access policy and a rule to quickly get you started. For simple use cases, this default Custom Authorization Server is usually all that you need.
```

Comment:
It doesn't feel right to use "default" as both a name and an adjective here. It seems ok to just say `For simple use cases, this Custom Authorization Server is...`

Reply:
@susanharper-okta: Heading of this section can be `pre-configured Custom Authorization Server` or `Out of the Box Custom Authorization Server`